pom.xml

```diff
@@ -179,7 +179,6 @@
     <joda.version>2.9.3</joda.version>
     <jodd.version>3.5.2</jodd.version>
     <jsr305.version>3.0.0</jsr305.version>
-    <libthrift.version>0.9.3</libthrift.version>
```

Comment:
@srowen . Can we keep this? For example, `jsr305.version` (at above line) is also used once but we have it.

Reply:
Sure that's fine. Normally I'd say it's just complicating the build but it also gives others a hook to change this version without maintaining a modification to the build file